src/pages/bioclimatic/index.js

```diff
@@ -146,38 +146,54 @@ const Container = () => {
     parsedScenarios
   };
 
-  const getChartConfig = unit => ({
-    lines: [
-      {
-        key: 'value',
-        color: styles.colorPink
-      }
-    ],
-    areas: [
-      {
-        key: 'value',
-        color: styles.colorPink
-      }
-    ],
-    yAxis: {
-      customTick: true,
-      tickSize: 0,
-      axisLine: false,
-      content: <Label value={unit} position="insideTop" dx={6} dy={-40} fill="#222" />
-    },
-    xAxis: {
-      customTick: true,
-      tickSize: 0,
-      padding: { left: -30, right: -30 }
-    },
-    grid: {
-      vertical: false
-    },
-    composedChart: {
-      margin: { top: 40, right: 40, left: 0, bottom: 0 }
-    },
-    height: 300
-  });
+  const getChartConfig = (unit, range) => {
+    let ticks;
+    if (range[1] - range[0] <= 1) ticks = [+range[0], +range[1]];
+    else if (range[1] - range[0] <= 2)
+      ticks = [+range[0], range[1] - (range[1] - range[0]) / 2, +range[1]];
+    else
+      ticks = [
+        +range[0],
+        range[1] - (range[1] - range[0]) * 0.66,
+        range[1] - (range[1] - range[0]) * 0.33,
+        +range[1]
+      ];
+    console.log(range, ticks);
```

Comment:
Please remove this little fellow from there ;)

Reply:
f$&@, I think we should treat `console.log`s as errors, not warnings. Sometimes if there's an error I see the warning too (in the pre-commit), but some other times I miss them.

Reply:
You're right, it's annoying. I created [a new PR](https://github.com/Vizzuality/copernicus-forest/pull/26) with a solution, thanks for proposing this :+1: